.golangci.yml

```diff
@@ -20,6 +20,11 @@ run:
     - integration
 
 linters-settings:
+  custom:
+    ll:
+      type: "module"
+      description: "Custom lll linter with log line exclusion."
```

Comment:
Should we customize the regular expression to `^\s*.*(L|l)og\.(Trace|Debug|Info|Warn|Error|Critical)S\(` to avoid false positives?

Reply:
cool yeah good idea!